pkg/protoutil/mask.go

```diff
@@ -0,0 +1,31 @@
+package protoutil
+
+import (
+	"github.com/mennanov/fmutils"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
+)
+
+// MergeAnyWithFieldMask merges the data in src with the data in dst,
+// but only the fields identified by the given mask.
+func MergeAnyWithFieldMask(dst, src *anypb.Any, mask *fieldmaskpb.FieldMask) *anypb.Any {
```

Comment:
should we really mask errors? it should probably be OK for a Put request to fail instead.

Reply:
I prefer it the way it was but I'll return the errors.